code/index.html

```diff
@@ -1,22 +1,163 @@
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./tiktok.jpeg" 
+        alt="Image of tiktok text written by a typwriter.">
+        <article>
+          <a href="">Crossword</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Fanny Svensson</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./grafitti.jpg" 
+        alt="Colorful grafitti.">
+        <article>
+          <a href="#">Goings On</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Foodlaura DeFoodie</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./paintings-at-museum.jpg" alt="Museum with paintings.">
+        <article>
+          <a href="#">Fiction & Poetry</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Robert Robertsson</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./troopers-jon-tyson-eIhH7RTlTZA-unsplash.jpg" 
+        alt="Image of storm troopers and Anonymous.">
+        <article>
+          <a href="#">Books & Culture</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Ticky DeFan.</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./tiktok.jpeg" 
+        alt="Image of tiktok text written by a typwriter.">
+        <article>
+          <a href="">Crossword</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Fanny Svensson</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./grafitti.jpg" 
+        alt="Colorful grafitti.">
+        <article>
+          <a href="#">Goings On</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Foodlaura DeFoodie</i></b>
+          </p>
+        </article>
+      </div>
+      <div class="small-news-card">
+        <img src="./paintings-at-museum.jpg" alt="Museum with paintings.">
+        <article>
+          <a href="#">Fiction & Poetry</a>
+          <h2>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit
+          </h2>
+          <p>
+            By <b><i>Robert Robertsson</i></b>
+          </p>
+        </article>
+      </div>
     </section>
-  </body>
-</html>
+  </main>
+  <footer>
+    <p><b>.rebkova.</b>
+      <br>The New Forker : The New Yorker magazine mock-up. 
```

Comment:
We should avoid `br` tags as much as we can - you could wrap `.rebkova.` text inside any element with default `display: block` and result would be the same :)